src/iop/Permutohedral.h

```diff
@@ -73,7 +73,7 @@ template <int KD, int VD> class HashTablePermutohedral
       for(int i = 0; i < KD; i++)
```

Comment:
This function is highly non-portable as it depends on the bit with of `size_t` and `unsigned`.